src/main/java/hudson/plugins/xvnc/Xvnc.java

```diff
@@ -92,7 +95,7 @@ public void setUp(Context context, Run<?, ?> build, FilePath workspace, Launcher
 
         String cmd = Util.nullify(DESCRIPTOR.xvnc);
         if (cmd == null) {
-            cmd = "vncserver :$DISPLAY_NUMBER -localhost -nolisten tcp";
+            cmd = "vncserver :$DISPLAY_NUMBER -localhost -nolisten tcp " + (this.additionalArgs != null ? this.additionalArgs : "");
```

Comment:
I am not totally sure, whether I understood the descriptor managed by the framework. However, I added a solution proposal. Otherwise, you have to guide me to the solution as I have not written a jenkins plugin before.